### PR TITLE
Content length to synclog

### PR DESCRIPTION
**Pull request by mjarco**

### CODE REVIEW

File: httphandler/response_merger.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ func (rd *responseMerger) synclog(r, successfulTup *transport.ReqResErrTuple) {
 		successfulTup.Req.URL.Path,
 		successfulTup.Req.Host,
 		r.Req.Header.Get("User-Agent"),
+		successfulTup.Res.Header.Get("Content-Length"),
```
Comment (janisz) [inline httphandler/response_merger.go]: What if header is not provided? Do we need a fallback?

File: httphandler/log.go [hunk 2]
```diff
@@ -57,33 +57,35 @@ func ScanCSVAccessLogMessage(csvstr string) (AccessMessageData, error) {
...
-	UserAgent   string `json:"useragent"`
-	ErrorMsg    string `json:"error"`
-	ReqID       string `json:"reqID"`
-	Time        string `json:"ts"`
+	Method        string `json:"method"`
+	FailedHost    string `json:"failedhost"`
+	Path          string `json:"path"`
+	SuccessHost   string `json:"successhost"`
+	UserAgent     string `json:"useragent"`
+	ContentLength string `json:"content-length"`
```
Comment (janisz) [inline httphandler/log.go]: Why `string` not `uint`?

File: httphandler/response_merger.go (hunk 1 above)
Comment (mjarco) [inline httphandler/response_merger.go]: yep. fixed

File: httphandler/log.go (hunk 2 above)
Comment (mjarco) [inline httphandler/log.go]: will be int64 negative number will indicate header was missing

File: httphandler/log.go [hunk 3]
```diff
@@ -57,33 +57,35 @@ func ScanCSVAccessLogMessage(csvstr string) (AccessMessageData, error) {
...
-	UserAgent   string `json:"useragent"`
-	ErrorMsg    string `json:"error"`
-	ReqID       string `json:"reqID"`
-	Time        string `json:"ts"`
+	Method        string `json:"method"`
+	FailedHost    string `json:"failedhost"`
+	Path          string `json:"path"`
+	SuccessHost   string `json:"successhost"`
+	UserAgent     string `json:"useragent"`
+	ContentLength int64  `json:"content-length"`
```
Comment (janisz) [inline httphandler/log.go]: Comment about negative value meaning would be nice.

**Pull request merged**